### DISCUSSION TITLE
Trouble installing on Win 11

### DISCUSSION THREAD

**Discussion by SorceressHeart** [nianeyna/ao3downloader#177] [Q&A]
![Screenshot 2024-12-01 163233](https://github.com/user-attachments/assets/a3b219e8-a4d4-4819-b7ee-55ae5a8ea919)

I'm using a new laptop which runs on windows 11. I followed instructions and this is what pops up when I try to run the command.  I think I had the latest python installed, uninstalled that, then installed version 3.11 which worked on my previous win 10 laptop. Any suggestions? Thanks.

**Comment by nianeyna** [comment]
❗ Checkpoint! ❗

Have you done ALL of the troubleshooting steps in the README? (The README is the explanatory text on the landing page for ao3downloader. You can find the Troubleshooting section using the Table of Contents at the beginning of the README.)

If you have done all the troubleshooting steps, please tell me what the results were. If you haven't, please do them to the best of your ability, and tell me what the results were. ✨ 

❗ Thank you! ❗

**Reply by SorceressHeart** [reply]
Ok. I think I have to reset my whole laptop first and transfer old stuff first, so it might take a few days, sorry. Thanks for your patience.

**Reply by SorceressHeart** [reply]
Hi! OK, so I think the issue was me trying to use the downloader from my old laptop. I downloaded Python 3.11.4  and did a new install of the download. One thing of note is that the new install did not have a 'downloads' folder, so I copied the old one from my old laptop. I think some files might be missing, but that's an issue with my transfer and not the downloader. 

Thanks for your patience with me. I will share if there are any further issues. Thanks again!

**Reply by nianeyna** [reply]
Glad you worked it out! The downloads folder being missing on a fresh copy of the script is expected, though - it's not created until you actually download something

**Reply by SorceressHeart** [reply]
Ah, makes sense. Thanks 😊
